### PR TITLE
[ROAD-281] Fix display messages on main panel

### PR DESCRIPTION
**Pull request by aldanchenko**
Fix display messages on main panel. Like "Run scan", "Select issue".

Screenshot 2021-09-18 at 11 40 16

### CODE REVIEW

File: Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/RunScanState.cs [hunk 1]
```diff
@@ -37,9 +35,9 @@ public override void DisplayComponents()
             {
                 await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-                this.ToolWindowControl.noVulnerabilitiesAddedMessageGrid.Visibility = Visibility.Visible;
+                this.ToolWindowControl.messagePanel.Visibility = Visibility.Visible;
 
-                this.ToolWindowControl.resultsGrid.Visibility = Visibility.Visible;
+                this.ToolWindowControl.messagePanel.RunScanMessage();
```
Comment (michelkaporin) [inline Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/RunScanState.cs:40]: RunScanMessage method intention is not clear from its signature, consider renaming here. For instance, `ShowRunScanMessage()`

File: Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/ScanResultsState.cs [hunk 2]
```diff
@@ -20,9 +20,9 @@ public class ScanResultsState : ToolWindowState
         {
             await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
-            this.ToolWindowControl.selectIssueMessageGrid.Visibility = Visibility.Visible;
+            this.ToolWindowControl.messagePanel.SelectIssueMessage();
```
Comment (michelkaporin) [inline Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/ScanResultsState.cs:23]: Method intention is not clear

File: Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/RunScanState.cs (hunk 1 above)
Comment (aldanchenko) [reply Snyk.VisualStudio.Extension/Snyk/VisualStudio/Extension/UI/Toolwindow/RunScanState.cs]: @michelkaporin thank you! Good point.

**Comment by aldanchenko** [comment]
@michelkaporin I renamed method names. Thank you!

**Review by michelkaporin — APPROVED** [review]
LGTM

**Pull request merged**